static/app/views/issueDetails/streamline/eventDetailsHeader.tsx

```diff
@@ -82,18 +83,25 @@ export function EventDetailsHeader({group, event, project}: EventDetailsHeaderPr
             </Flex>
           </Fragment>
         )}
-        {issueTypeConfig.header.graph.enabled && (
-          <GraphSection>
-            <EventGraph event={event} group={group} style={{flex: 1}} />
-            {issueTypeConfig.header.tagDistribution.enabled && (
-              <IssueTagsPreview
-                groupId={group.id}
-                environments={environments}
-                project={project}
-              />
-            )}
-          </GraphSection>
-        )}
+        {issueTypeConfig.header.graph.enabled &&
+          issueTypeConfig.header.graph.type === 'discover-events' && (
+            <GraphSection>
+              <EventGraph event={event} group={group} style={{flex: 1}} />
+              {issueTypeConfig.header.tagDistribution.enabled && (
+                <IssueTagsPreview
+                  groupId={group.id}
+                  environments={environments}
+                  project={project}
+                />
+              )}
+            </GraphSection>
+          )}
+        {issueTypeConfig.header.graph.enabled &&
+          issueTypeConfig.header.graph.type === 'detector-history' && (
+            <MetricChartSection>
+              <MetricIssueChart group={group} project={project} event={event} />
+            </MetricChartSection>
+          )}
```

Comment:
I would move this inside the existing graph section to allow the tag distribution to still be optional in case they ever get added to metric issues. GH suggested commits are gonna have bad formatting so lemme pseudo code it:

```
config.graph.enabled && <GraphSection>
  config.graph == 'events' && <EventGraph />
  config.graph == 'metrics' && <MetricChart />
  config.header.tags  && <TagDistribution />
</GraphSection>
```

You can even get rid of `<MetricChartSection>` by adding some prop to `GraphSection` if you want:


```js
const GraphSection = styled('div')<{isMetricChart: boolean}>`
  padding-right: ${p => p.isMetricChart ? space(1.5) : 0};
`
```